mono/mini/mini-generic-sharing.c

```diff
@@ -65,6 +65,7 @@ partial_sharing_supported (void)
 {
 	if (!ALLOW_PARTIAL_SHARING)
 		return FALSE;
+	return TRUE;
```

Comment:
Was this intentional? This is going to make all the logic past this dead code.

Reply:
Fixed.